js/network.js

```diff
@@ -0,0 +1,56 @@
+'use strict';
+window.network = (function () {
+  return {
+    receive: function (onLoad, onError, errBtnHandler) {
+      var receiveDataUrl = 'https://js.dump.academy/keksobooking/data';
+      var xhr = new XMLHttpRequest();
+
+      function xhrReceiveHandler(event) {
+        if (event.target.status === 200) {
+          onLoad(event.target.responseText);
+        } else {
+          onError(event.target.status, errBtnHandler);
+        }
+      }
+
+      function xhrErrorHandler(event) {
+        onError(event.target.status, errBtnHandler);
+      }
+
+      function xhrTimeoutHandler(event) {
+        onError(event.target.status, errBtnHandler);
+      }
+
+      xhr.addEventListener('load', xhrReceiveHandler);
+      xhr.addEventListener('error', xhrErrorHandler);
+      xhr.addEventListener('timeout', xhrTimeoutHandler);
+
+      xhr.timeout = 10000;
```

Comment:
Why did you decide to use such a long timeout? 10 seconds?

Reply:
done